ci/verify_bundle.py

```diff
@@ -310,16 +193,51 @@ def verify_torchscript(
         print("Provided TorchScript module is verified correctly.")
 
 
+def verify_bundle_properties(model_path: str, bundle: str):
+    """
+    This function is used to verify the bundle properties.
+
+    """
+    bundle_path = os.path.join(model_path, bundle)
+    meta_file = os.path.join(bundle_path, "configs/metadata.json")
+    metadata = get_json_dict(meta_file)
+    # Since lack of data, train and evaluate config properties can checked in unit tests
+    # In this file, only inference config properties will be checked
+    for workflow_type in ["inference"]:
+        config_name = _find_bundle_file(os.path.join(bundle_path, "configs"), workflow_type)
+        if config_name is not None:
+            config_file = os.path.join(bundle_path, f"configs/{config_name}")
+            meta_file = os.path.join(bundle_path, "configs/metadata.json")
+            if "supported_apps" in metadata:
+                # TODO: get properties for each app.
+                pass
+            else:
+                properties_path = None
+                # if not specified, use the default properties
+                print("workflow_type: ", workflow_type)
+                workflow = create_workflow(
+                    workflow_type=workflow_type,
+                    bundle_root=bundle_path,
+                    config_file=config_file,
+                    logging_file=os.path.join(bundle_path, "configs/logging.conf"),
+                    meta_file=meta_file,
+                    properties_path=properties_path,
+                )
+            check_result = workflow.check_properties()
```

Comment:
If "supported_apps" is in `metadata` then `workflow` won't be created.

Reply:
Thanks @ericspod , I changed here